framework/src/modules/interoperability/constants.ts

```diff
@@ -17,10 +17,18 @@ export const MODULE_NAME_INTEROPERABILITY = 'interoperability';
 
 // General constants
 export const MAINCHAIN_ID = 1;
+export const MAINCHAIN_NAME = 'lisk-mainchain';
+export const TAG_CHAIN_REG_MESSAGE = 'LSK_CHAIN_REGISTRATION';
 export const LIVENESS_LIMIT = 2592000; // 30*24*3600
 export const MAX_CCM_SIZE = 10240;
 export const EMPTY_FEE_ADDRESS = Buffer.alloc(0);
 export const EMPTY_BYTES = Buffer.alloc(0);
+export const REGISTRATION_FEE = 1000000000;
```

Comment:
In the LIP a `uint64` type is used for this variable although the current value fits in `uint32`. I think it'd be nice either to clarify the maximum allowed value of this constant or just use type `BigInt` since `JS number` supports maximum 53 bits.